src/components/Templates.vue

```diff
@@ -1,35 +1,33 @@
 <template>
   <div data-test="templates">
     <div class="templates-bar">
-      <TopBar :tabs="tabs">
-        <template v-slot:tool>
-          <button class="tool-btn" @click="importTemplate">
-            <font-awesome-icon class="icon" icon="file-import" />
-            Import template
-          </button>
-        </template>
-      </TopBar>
+      <p class="title">
+        Data Request Templates <span class="templates-number">{{ templates.length }}</span>
+      </p>
+      <el-button class="import-btn" @click="importTemplate" type="primary">
+        <font-awesome-icon class="icon" icon="file-import" />
+        Import template
+      </el-button>
     </div>
     <div class="centered">
       <div v-if="Object.entries(paginatedItems)" class="container-templates">
-        <div v-show="currentPage === 1" class="add">
-          <router-link to="/request/editor">
-            <img
-              data-test="create-template"
-              @click="createTemplate"
-              class="add-btn"
-              src="@/resources/svg/add.svg"
-            />
-          </router-link>
+        <div
```

Comment:
All templates and create template button should have the same size

Reply:
They have the same size
<img width="1440" alt="Captura de pantalla 2020-05-07 a las 14 59 25" src="https://user-images.githubusercontent.com/16032491/81297368-6cd8ae80-9073-11ea-9fe2-69fe420216ab.png">
<img width="1440" alt="Captura de pantalla 2020-05-07 a las 14 59 32" src="https://user-images.githubusercontent.com/16032491/81297371-706c3580-9073-11ea-9b81-0063035bc9bc.png">